Demo/Form1.cs

```diff
@@ -30,6 +34,8 @@ public Form1()
 			incrementalSearcher1.FindReplace = MyFindReplace;
 
 			findAllResultsPanel1.Scintilla = scintilla1;
+
+			InitNumberMargin();
```

Comment:
Could this empty margin be initialized / cleared when toggling the custom line numbers?
![image](https://github.com/VPKSoft/ScintillaNET-FindReplaceDialog/assets/40712699/c276865d-e744-4183-99f1-559376510ce3)